setup.py

```diff
@@ -1,14 +1,11 @@
-import sys
-version = sys.argv[1]
-del sys.argv[1]
 import setuptools
 
 with open("README.md", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="stock-indicators",
-    version=version,
+    version="0.7.0",
```

Comment:
Yeah, it would be better if we can commit from Pipelines for that replacement, otherwise, we should make a commit  only for changing version num.

Reply:
Though plus sign convention can keep commit uniqueness, that creates too many dummy commits. AND it is not accepted, according to [PEP-440](https://peps.python.org/pep-0440/#semantic-versioning)

How about adding it to CD pipeline ? Download artifact(wheel), install it, fetch version number from `pip list` and then, makes commit with replaced setup.py.

Reply:
I'm missing context on this one.  I thought our versioning, build, deploy were all working okay.  What's the problem we're trying to solve?

Reply:
This is for the users who want to build manually from the repo. Our setup.py is receiving semantic version number from CLI arguments. And it cause the problem in Docker like #197, user have to pass fake version number to setup.py to build and install. That also means `pip -e Stock.Indicators.Python/`, which installs from source directory using setup.py, will not work because there's no way to pass version number.

Reply:
I see.  Maybe we can add some conditional expressions that reference the GitHub or Azure Pipelines meta-data to do it one way, then an alt thing for external uses?  I’d like to keep our CI/CD automated as it will keep us consistent.  Manual steps increase the risk of versioning errors.

Reply:
I think it's about where we deal with it, build.yml or setup.py. Then I'll try to handle it in setup.py.